frontstage/views/passwords/forgot_password.py

```diff
@@ -31,29 +32,27 @@ def post_forgot_password():
 
     if form.validate():
         email_address = request.form.get('email_address')
-        post_data = {"username": email_address}
-        response = api_call('POST', app.config['REQUEST_PASSWORD_CHANGE'], json=post_data)
 
-        # If we receive a 401 parse the error message to display the correct reason why
-        if response.status_code == 401:
-            error_json = json.loads(response.text).get('error')
-            error_message = error_json.get('data', {}).get('detail')
+        try:
+            oauth_controller.check_account_valid(email_address)
+        except OAuth2Error as exc:
+            error_message = exc.message
             logger.info(error_message=error_message)
```

Comment:
Is there a way this can be a bit more specific so that is can be traced to this part of the code from the logs?

Reply:
Not without including the email address. Is the request ID present in the splunk logs?